community/cypher/acceptance/src/test/scala/org/neo4j/internal/cypher/acceptance/MatchAcceptanceTest.scala

```diff
@@ -27,7 +27,7 @@ import scala.collection.JavaConverters._
 
 class MatchAcceptanceTest extends ExecutionEngineFunSuite with QueryStatisticsTestSupport with NewPlannerTestSupport {
 
-  test("path query should return results in written order") {
+  ignore("path query should return results in written order") {
```

Comment:
I think these `ignore` changes are good to have when knowing which tests have been ported and which haven't, but we need to recall to revert these changes back before merging this.